wholecell/utils/build_ode.py

```diff
@@ -26,22 +26,30 @@ def build_functions(arguments, expression):
 		a lambda function(arguments)
 		a Numba Dispatcher function(arguments)
 	"""
-	f = eval('lambda {}: {}'.format(arguments, expression), {'np': np}, {})
+	local_dict = {}
```

Comment:
This'll need to be `local_dict: dict[str, Any] = {}` to satisfy mypy, and import Any from typing.